src/components/common/Alert/Alert.tsx

```diff
@@ -0,0 +1,33 @@
+import { useStorage } from '@faststore/sdk'
+import { BellRinging as BellRingingIcon } from 'phosphor-react'
+import React, { useCallback } from 'react'
+import UIAlert from 'src/components/ui/Alert'
+import { mark } from 'src/sdk/tests/mark'
+import type { PropsWithChildren } from 'react'
+
+function Alert({ children }: PropsWithChildren<unknown>) {
+  const [displayAlert, setDisplayAlert] = useStorage('alert', true)
+
+  const onAlertClose = useCallback(
+    () => setDisplayAlert(false),
+    [setDisplayAlert]
+  )
+
+  if (displayAlert === false) {
+    return null
+  }
+
+  return (
+    <UIAlert
+      icon={<BellRingingIcon size={24} />}
+      dismissible
+      onClose={onAlertClose}
+    >
+      {children}
+    </UIAlert>
+  )
```

Comment:
```suggestion
  if (displayAlert === false) {
    return null
  }

  const onAlertClose =  () => setDisplayAlert(false)
  
  return (
    <UIAlert
      icon={<BellRingingIcon size={24} />}
      dismissible
      onClose={onAlertClose}
    >
      {children}
    </UIAlert>
  )
```

useCallback has a "price" that can be avoided.

Reply:
Yes, but it's the default path for the anonymous user, so we kind of always pay this price on lighthouse. It's better paying this and using `useCallback`.